runtime/src/vm/opcode.rs

```diff
@@ -1828,10 +1828,38 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
   });
   cpu!("poparr", |args, hand_mem, _, _| {
     let last = hand_mem.pop_fractal(args[0]);
+    hand_mem.new_fractal(args[2]);
```

Comment:
Non blocking, but I wonder if it would be possible to DRY some of the Result/Maybe logic from opcodes

Reply:
Oh, DRYing up a lot of stuff is on my TODO list. Most of the duplication surrounds handling Pascal-style strings in HandlerMemory objects. I hope to centralize it so we can eventually find a more efficient way to do it.

The other part of DRYing it up is with Result and Maybe types being replicated within the opcodes, yes, so making helper functions for that is probably good. But DRY also makes it harder to "do something like this, but slightly different" so I prefer to duplicate for a bit and then DRY it up afterwards once the patterns emerge.